src/main/java/org/mastodon/mamut/ProjectManager.java

```diff
@@ -857,14 +863,16 @@ private void saveGUI( final ProjectWriter writer ) throws FileNotFoundException,
 		final Element guiRoot = new Element( GUI_TAG );
 		guiRoot.setAttribute( MAMUTPROJECT_VERSION_ATTRIBUTE_NAME, MAMUTPROJECT_VERSION_ATTRIBUTE_CURRENT );
 		final Element windows = new Element( WINDOWS_TAG );
-		windowManager.forEachView( ( view ) -> windows.addContent(
-				MamutViewStateSerialization.toXml( view ) ) );
-		windowManager.forEachBranchView( ( view ) -> windows.addContent(
-				MamutViewStateSerialization.toXml( view ) ) );
```

Comment:
It's hard to spot what actually changed, since there are so many lines just reformatted could you undo the formatting please!